src/engine/ExportQueryExecutionTrees.h

```diff
@@ -37,7 +37,8 @@ class ExportQueryExecutionTrees {
   // lazily computes the serialized result in large chunks of bytes.
   static cppcoro::generator<std::string> computeResultAsStream(
```

Comment:
As this is the only function left, we can simply call it ` computeResult` (no ` AsStream` needed, as everything is lazy.